compiler/src/dotty/tools/dotc/printing/SyntaxHighlighting.scala

```diff
@@ -107,9 +107,11 @@ object SyntaxHighlighting {
             case tree: ValOrDefDef =>
               highlightAnnotations(tree)
               highlightPosition(tree.nameSpan, ValDefColor)
+              highlightPosition(tree.endSpan, ValDefColor)
```

Comment:
Could we add a test in `SyntaxHighlightingTests.scala` with a test using an `end` to test this?